### ISSUE TITLE
Install Helm Diff plugin to have better idempotency check results

### ISSUE THREAD

**Issue by ricsanfre** [ricsanfre/pi-cluster#21]
## Issue Description

Whenever trying to redeploy/upgrade a helm char using ansible `kubernetes.core.helm` module the following warning appears

[WARNING]: The default idempotency check can fail to report changes in certain cases. Install helm diff for
better results.

[helm diff](https://github.com/databus23/helm-diff) should be installed to do a better indempotency check and avoid this warning.

**Comment by ricsanfre** [comment]
helm-diff does not have support yet of ARM 64 architectures.

Plugin installation launch an error:
```console
ansible@node1:~$ helm plugin install https://github.com/databus23/helm-diff
No prebuild binary for linux-arm64.
Failed to install helm-diff
        For support, go to https://github.com/databus23/helm-diff.
Error: plugin install hook for "diff" exited with error
```

There is an open PR to provide ARM support (https://github.com/databus23/helm-diff/pull/267), but it is not resolved yet.


When that PR is solved there is an easy way to install the plugin using `kuberentes.core.helm_plugin` ansible module

Add following task to `k3s-deploy.yml`  `pre-tasks` section:

```yml
    # Install Helm diff plugin to have a better idempotence check
    - name: Intall Helm Plugin
      kubernetes.core.helm_plugin:
        plugin_path: "https://github.com/databus23/helm-diff"
        state: present
      tags: ['always']
```

**Comment by ricsanfre** [comment]
PR has been resolved and new release v3.2.0 provides ARM64 bit architecture support. See v3.2.0 [release notes](https://github.com/databus23/helm-diff/releases/tag/v3.2.0).
